scc/canary-deployment.hbs.md

```diff
@@ -0,0 +1,73 @@
+# Use Canary deployment with Contour and Carvel Packages for Supply Chain Choreographer (alpha)
+
+Canary deployment is an incremental approach to releasing an application where traffic is divided between the existing deployed version and a new version.
+It involves introducing the new version to a smaller group of users before extending it to the entire user base.
+This topic outlines how to automate canary releases using Contour ingress controller and Flagger with Packages and PackageInstalls.
+
+## <a id="prereqs"></a> Prerequisites
+
+To use canary deployment, you must complete the following prerequisites:
+
+- Complete the prerequisites in [Configure and deploy to multiple environments with custom parameters](config-deploy-multi-env.hbs.md).
+- Configure your Supply Chain to output Carvel Packages. See [Output Carvel Packages from your Supply Chain](carvel-package-supply-chain.hbs.md).
+- Deploy Carvel Packages using the tool of your choice:
+  - [Deploy Carvel Packages using Carvel App CR](delivery-with-carvel-app.hbs.md)
+  - [Deploy Carvel Packages using FluxCD Kustomization](delivery-with-flux.hbs.md)
+  - [Deploy Carvel Packages using ArgoCD](delivery-with-argo.hbs.md)
+- Install Flagger on your Kubernetes cluster. See [Flagger Install on Kubernetes](https://docs.flagger.app/install/flagger-install-on-kubernetes).
+
+## <a id="instructions"></a> How to use Contour ingress controller and Flagger to create a canary release
+
+Flagger is a tool that facilitates a controlled process of shifting traffic to a canary deployment while monitoring essential performance metrics
+such as the success rate of HTTP requests, average request duration, and the health of the application's pods.
+By analyzing these key indicators, Flagger determines whether to promote the canary deployment to a wider audience or
+abort it if any issues arise.
+
+For creating canary releases, you can refer to the guide available at [Contour Canary Deployments](https://docs.flagger.app/tutorials/contour-progressive-delivery).
+In summary, the process explained in the guide above involves the following steps:
+
+1. Install Flagger on your Kubernetes cluster.
+
+2. Add the label `app.kubernetes.io/name` to your `Workload`.
```

Comment:
Is the label automatically applied to the underlying K8s deployment? Just to confirm

Reply:
Yes, I confirmed the label gets applied to the k8s deployment 👍🏼